homework/index.js

```diff
@@ -23,24 +23,147 @@
       const value = options[key];
       if (key === 'text') {
         elem.textContent = value;
+      } else if (key === 'html') {
+        elem.innerHTML = value;
```

Comment:
By adding an option to set the `innerHTML` this function has now become ambiguous in the sense that it is no longer clear which of the two element-creation methods is to be used: `document.createElement()` or `innerHTML`.

Reply:
I reverted the createAndAppend function to the unmodified version. You are right that my version was going outside of its concept somehow.